recipe_batches/recipe_batches.py

```diff
@@ -2,13 +2,33 @@
 
 import math
 
+
 def recipe_batches(recipe, ingredients):
-  pass 
+    """Calculates the largest number of batches of a recipe possible."""
+    low_multiple = None
+    # Determine which ingredient has the least on hand, in terms of amount
+    # needed to make the recipe.
+    for ingredient in recipe.keys():
```

Comment:
there's a cool Python feature I just learned about, all(), that can trim this down to a single line. check it out!

https://www.programiz.com/python-programming/methods/built-in/all